repo2docker/buildpacks/r.py

```diff
@@ -0,0 +1,156 @@
+import re
+import os
+import datetime
+
+from .python import PythonBuildPack
+
+class RBuildPack(PythonBuildPack):
+    """
+    Setup R for use with a repository
+
+    This sets up R + RStudio + IRKernel for a repository that contains:
+
+     1. A `runtime.txt` file with the text:
+
+        r-<year>-<month>-<date>
+
+        Where 'year', 'month' and 'date' refer to a specific
+        date snapshot of https://mran.microsoft.com/timemachine
+        from which libraries are to be installed.
+
+     2. An optional `install.R` file that will be executed at build time,
+        and can be used for installing packages from both MRAN and GitHub.
+
+    It currently sets up R from the ubuntu repository being used. This
```

Comment:
R from the ubuntu repository being used. I'm not sure what that means.

Reply:
Will clarify (it means R is installed via 'apt-get install r-base', rather than any of the methods available from https://cran.r-project.org/)

Reply:
OK that makes sense. Thanks @yuvipanda